ipaplatform/redhat/tasks.py

```diff
@@ -484,6 +485,33 @@ def configure_http_gssproxy_conf(self, ipaapi_user):
         os.chmod(paths.GSSPROXY_CONF, 0o600)
         self.restore_context(paths.GSSPROXY_CONF)
 
+    def configure_httpd_wsgi_conf(self):
+        """Configure WSGI for correct Python version"""
+        conf = paths.HTTPD_IPA_WSGI_MODULES_CONF
+        if sys.version_info > (3, 0):
+            wsgi_module = constants.MOD_WSGI_PYTHON3
+        else:
+            wsgi_module = constants.MOD_WSGI_PYTHON3
```

Comment:
should not it be `constants.MOD_WSGI_PYTHON2` here?

Reply:
yeah, got catch :)